### ISSUE TITLE
Warn mode removes audit unless user blocks

### ISSUE THREAD

**Issue by jgriff500** [MicrosoftDocs/microsoft-365-docs#8879]
[Enter feedback here]
It should be noted that "warn" mode removes audit events, unless the user selects to block the action -- then a block event is logged.    A better action for the product would be to have an option to audit when user unblocks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 96cedf2c-e6fa-bbfb-61f0-ab4bcaae1479
* Version Independent ID: 45ca9ffb-55e9-7721-3433-017a513a59be
* Content: [Use attack surface reduction rules to prevent malware infection](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/attack-surface-reduction?view=o365-worldwide&source=docs)
* Content Source: [microsoft-365/security/defender-endpoint/attack-surface-reduction.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/attack-surface-reduction.md)
* Product: **m365-security**
* Technology: **mde**
* GitHub Login: @jweston-1
* Microsoft Alias: **v-jweston**

**Comment by jweston-1** [comment]
I have forwarded the customer's comments to the product team for review and consideration.

**Comment by jweston-1** [comment]
I have submitted the customer's comments and suggestions to the product team for review and consideration. I will act on their recommendation once received.

**Comment by yogkumgit** [comment]
@jweston-1 Please comment, if you an update. Thanks

**Comment by jweston-1** [comment]
Per the product team response from Sujit (in separate thread of this mail), Yogesh, Please resolve as "won't fix."
